include/ck_tile/host/check_err.hpp

```diff
@@ -108,7 +110,7 @@ double get_absolute_threshold(const double max_possible_num, const int number_of
     }
     double midway_error = std::max(compute_error, output_error);
 
-    static_assert(is_any_of<AccDataType, F8, F16, BF16, F32, I8, I32, int>::value,
+    static_assert(is_any_of<AccDataType, F8, BF8, F16, BF16, F32, I8, I32, int>::value,
```

Comment:
Do we have numeric_traits for bf8?

Reply:
Yes. it is defined in ck_tile/core/numeric/float8.hpp.